tests/Flywheel/FlywheelTest.js

```diff
@@ -761,8 +773,8 @@ describe('Flywheel', () => {
 
     it('should stop bankrupt accounts from claiming', async () => {
       // claiming venus will fail
-      const venusRemaining = bnbUnsigned(100e18);
-      const accruedAmt = bnbUnsigned(10e18);
+      const venusRemaining = bnbUnsigned(2e12);
+      const accruedAmt = bnbUnsigned(1e12);
```

Comment:
Why do we need to change these values?

Reply:
@kkirka values were copied accidentally, it was meant for other test
changes are reverted